cornac/models/next_basket_recommender.py

```diff
@@ -0,0 +1,325 @@
+# Copyright 2018 The Cornac Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+
+import os
+import copy
+import inspect
+import pickle
+from glob import glob
+from datetime import datetime
+
+import numpy as np
+
+from ..exception import ScoreException
+
+
+class NextBasketRecommender:
```

Comment:
It should inherit from Recommender to avoid redundancy and maintain consistency

Reply:
We can add this class in the `recommender.py` file (it's a base file for all types of recommender abstractions). Don't need to create a new file.